src/app/component/ProductActions/ProductActions.component.js

```diff
@@ -404,6 +405,19 @@ export class ProductActions extends PureComponent {
         );
     }
 
+    allPricesAreSame() {
+        const {
+            product: { variants }
+        } = this.props;
+
+        if (variants && variants.length > 0) {
+            return !variants.some((variant) => JSON.stringify(variant.price_range.minimum_price.final_price)
+                !== JSON.stringify(variants[0].price_range.minimum_price.final_price));
```

Comment:
Do not break expressions & operators on to the next line :) There must be a better way!